src/the-nest/annotations.py

```diff
@@ -0,0 +1,36 @@
+x: int
```

Comment:
Related to the other suggestion.
```suggestion
from textwrap import wrap

x: int
```